javers-persistence-sql/src/main/java/org/javers/repository/sql/ConnectionProvider.java

```diff
@@ -11,5 +12,5 @@
  */
 public interface ConnectionProvider {
     
-    Connection getConnection();
+    Connection getConnection() throws SQLException;
```

Comment:
Zobacz: https://github.com/javers/javers/pull/270/files#diff-fd1fe755fa8e5efed1dbf40306c77d0fR27

nie chciało mi się łapać wyjątku w tej anonimowej deklaracji klasy, tym bardziej, że we wszystkich użyciach tej metody w Javersie jest to opakowane w blok try cach z łapanym SqlException. Jeżeli chcemy to usuniemy to każdy user w java-style konfiguracji będzie musiał taką transformację wyjatku napisać własnoręcznie.